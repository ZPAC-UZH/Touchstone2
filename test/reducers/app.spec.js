import reducer from '../../app/scripts/store/app/AppReducer';
import {
  switchToWorkingMode,
  WORKING_MODE_SWITCHER_SUCCESS,
} from '../../app/scripts/store/app/AppActions';

describe('App', () => {
  let app = reducer(undefined, {});

  it('should return the initial state', () => {
    expect(reducer(app, {}))
      .toMatchSnapshot();
  });

  it(`should handle ${WORKING_MODE_SWITCHER_SUCCESS}`, () => {
    app = reducer(app, switchToWorkingMode('standard'));
    expect(app)
      .toMatchSnapshot();
  });
});
