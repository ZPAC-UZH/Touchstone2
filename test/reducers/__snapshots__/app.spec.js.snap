// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should handle WORKING_MODE_SWITCHER_SUCCESS 1`] = `
Object {
  "workingMode": "WORKING_MODE_STANDARD",
}
`;

exports[`App should return the initial state 1`] = `
Object {
  "workingMode": "WORKING_MODE_STANDARD",
}
`;
